openedx/features/smart_referral/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class MarketplaceConfig(AppConfig):
```

Comment:
Won't that be `SmartReferralConfig` as the app directory and below name field says

Reply:
Good catch @UsmanArbisoft